website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -93,43 +152,90 @@ Row placement determines the visual styling based on where the row is placed rel
 
 ### Density
 
-![Table cell density](/assets/components/table/table-density.png =798x*)
+![Table cell density](/assets/components/table/table-density.png)
 
 - We recommend using `medium` cell density by default.
 - If content is complex or a smaller data set (e.g., a Table of basic user data), `tall` cell density allows for more breathing room surrounding the content.
 - If content is largely string/text-based, `short` allows for more content to be displayed within the page.
 - While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
 
-### Content
+### Icon usage
 
-While we are not prescriptive about what goes into a cell, there are some best practices to consider:
+Icons used within cells can help differentiate content, highlight additional metadata, increase the hierarchy of a value, or otherwise _enhance_ the text or value it is paired with. Use the outlined icon style by default and if contrast against other icons is important, use the filled style.
 
-- We recommended keeping data within a column to one data type. Using more than one data type makes sorting almost impossible.
-- While it‘s possible to change the text style/color within a cell, we recommended only using Helios font styles.
+!!! Dont
 
-### Icon usage
+Don’t use an icon as the sole communication method within a cell, even if the icon is explicit, e.g., a brand or service icon.
 
-Icons nested within cells can help differentiate content, see status, and increase the hierarchy of a piece of data or object. Use the outlined icon style by default and if contrast against other icons is important, use the filled style.
+![Icon within a table without a label](/assets/components/table/icon-without-label.png)
+!!!
 
-Icons should rarely be used without a text label. A label helps reinforce the purpose and communication of the icon and can reduce ambiguity when expressing complex data.
+!!! Dont
 
-!!! Do
+Don’t use an icon to indicate the status of an object, row, or resource. Instead, use a [Badge](components/badge).
```

Comment:
I'm okay with leaving this in for now and listening to consumer feedback. That said, I wonder if we could soften the language slightly so that this is more of a "caution" than a "don't."

```suggestion
We don't recommend using an icon to indicate the status of an object, row, or resource. Instead, consider using a [Badge](components/badge).
```